app/controllers/api/v1/posters_controller.rb

```diff
@@ -40,4 +38,12 @@ def destroy
     def poster_params
         params.require(:poster).permit(:name, :description, :price, :year, :vintage, :img_url)
     end
+
+    def inject_meta_object(result_instance)
+        if result_instance.is_a?(ActiveRecord::Relation )
```

Comment:
This works technically, but It seems like a it could be refactored in the future to be less narrowly applicable